src/interpreter.jl

```diff
@@ -15,7 +15,7 @@ end
 Evaluates all examples and returns true iff all examples pass.
 Shortcircuits as soon as an example is found for which the program doesn't work. 
 """
-function evaluate_examples(tab::SymbolTable, expr::Any, examples::Vector{Data.Example})::Bool
+function test_examples(tab::SymbolTable, expr::Any, examples::Vector{Data.Example})::Bool
```

Comment:
This function is used in Search.jl in `search` in `search_procedure.jl`, so we should probably also change this usage after merging this.